auraed/src/runtime/cells/executable.rs

```diff
@@ -84,14 +85,15 @@ impl Executable {
                     info!("Unshare: cgroup");
                     namespaces_to_unshare.push(Namespace::Cgroup)
                 }
+
                 let command = command.unshare(&namespaces_to_unshare);
 
                 // Run 'pre_exec' hooks from the context of the soon-to-be launched child.
                 let command = {
                     let executable_name = self.name.clone();
                     unsafe {
                         command.pre_exec(move || {
-                            aurae_process_pre_exec(&executable_name)
+                            pre_exec(&executable_name, spec.clone())
```

Comment:
are we ignoring the potential error result here?

Reply:
We are not. `command.pre_exec` takes a function that must return an `io::Result<()>` (i.e., `std::result::Result<(), io::Error>`), which `pre_exec` (the inner one) does. If `pre_exec` returns an error, it will bubble up when calling `command.spawn`, and a few lines down we have `let child = command.spawn()?;`